tests/Syncromatics.Clients.Metro.Api.Tests/Syncromatics.Clients.Metro.Api.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>    
-    <TargetFrameworks>netstandard1.6;net46</TargetFrameworks>
+    <TargetFrameworks>net46</TargetFrameworks>
```

Comment:
Any reason not to go netstandard here?

Reply:
IIRC, there was an issue with the XUnit runner and netstandard. I haven't looked at that recently.

Reply:
Just looked at this again. The XUnit test runner can't load netstandard20 or netcoreapp20 assemblies because it's built against net46. The tests were only ever running the net46 build of the assembly, so I just removed the extraneous build.